views/components/entries/StatementSidePanel/StatementInputDialog/Header.ejs

```diff
@@ -78,4 +78,29 @@ function interpretButton() {
     }
 }
 
+function minimizeButton() {
+    return {
+        onClick: function(onClick){
+            $('#windowdown-link').on('click', onClick)
```

Comment:
Yes, I thought this also but the problem is that there's quite a few buttons that are very specific for specific views, so I actually removed your tooltip function because otherwise we have 2 different logics: one is based on the functionality and location in the interface (StatementsList / Graph) and another one is based on whether it's a tooltip or not. Perhaps we can look at it together later. I'm sure it'll be easy to change that. Plus the tooltip extra is not so big anyway.